src/Sylius/Bundle/ApiBundle/Resources/config/services/filters.xml

```diff
@@ -47,44 +47,44 @@
 <!--            <tag name="api_platform.filter" />-->
 <!--        </service>-->
 
-<!--        <service id="sylius.api.product_association_filter" parent="api_platform.doctrine.orm.search_filter" public="true">-->
-<!--            <argument type="collection">-->
-<!--                <argument key="owner.code">partial</argument>-->
-<!--                <argument key="type.code">partial</argument>-->
-<!--            </argument>-->
-<!--            <tag name="api_platform.filter" />-->
-<!--        </service>-->
+        <service id="sylius.api.product_association_filter" parent="api_platform.doctrine.orm.search_filter" public="true">
```

Comment:
There was a plan to start naming the filters in a new way since version 2.0, but I don't know if it is still valid. By using the prefix `sylius.api.order_filter` and making them only for one field.